src/RadixDlt.NetworkGateway.PostgresIntegration/LedgerExtension/AccountAuthorizedDepositorsProcessor.cs

```diff
@@ -93,10 +93,10 @@ internal class AccountAuthorizedDepositorsProcessor
     private readonly ChangeTracker<AccountAuthorizedDepositorsChangePointerLookup, AccountAuthorizedDepositorsChangePointer> _accountAuthorizedDepositorsChanges = new();
 
     private readonly Dictionary<long, AccountAuthorizedDepositorAggregateHistory> _mostRecentAuthorizedDepositorAggregates = new();
-    private readonly Dictionary<AccountAuthorizedDepositorsDbLookup, AccountAuthorizedDepositorHistory> _mostRecentAuthorizedDepositorEntries = new();
+    private readonly Dictionary<AccountAuthorizedDepositorsDbLookup, AccountAuthorizedDepositorEntryHistory> _mostRecentAuthorizedDepositorEntries = new();
 
     private readonly List<AccountAuthorizedDepositorAggregateHistory> _accountAuthorizedDepositorAggregatesToAdd = new();
-    private readonly List<AccountAuthorizedDepositorHistory> _accountAuthorizedDepositorHistoryToAdd = new();
+    private readonly List<AccountAuthorizedDepositorEntryHistory> _accountAuthorizedDepositorHistoryToAdd = new();
```

Comment:
Just a super minor thought: if we're already in the context of `Account...Processor` wouldn't it be easier to use shorter field names? Think: `_mostRecentAggregates`, `_aggregatesToAdd` etc.

Reply:
fixed.